src/controllers/EvidenceQuestionController.ts

```diff
@@ -33,13 +33,20 @@ const navigation: Navigation = {
     }
 };
 
+const changeModeAction = (req: Request, step: keyof NavigationControl) => {
```

Comment:
I don't think this is needed as the 'change' link on the Check Your Appeal page takes you back to the Upload page and then back again to the Check Your Appeal page